### DISCUSSION TITLE
Assign a url to css assets

### DISCUSSION THREAD

**Discussion by yishay-at-bay** [webpack/webpack#18403] [Q&A]
I want to have the output html file in **production** with a link tag for the css asset that looks something like that
`<link href="https://some-cdn-url/main.026ea848882c3b721917.css" rel="stylesheet">`
where the `main.026ea848882c3b721917.css` is the regular hashed asset name and `https://some-cdn-url` is some address that holds the css (I will take care to the part that puts the asset there)

I'm using MiniCssExtractPlugin in production so I tried using the filename option to do that
But with the filename option no matter what value I'm giving it, the output in the html file will always start with a slash. It also translates some of the chars.
So if I put

```
new MiniCssExtractPlugin({
  filename: 'https://some-cdn-url/[name].[contenthash].css',
}),
```

I get
`<link href="/https%3A//some-cdn-url/main.026ea848882c3b721917.css" rel="stylesheet">`

With `/` at the beginning and the `:` turns into `%3A`
Is there a way, with some plugin or some other tool, that I can create the output correctly so the browser will know to request the css asset from the right address?

**Comment by alexander-akait** [comment]
Answered in `mini-css-extract-plugin` repo, feel free to feedback
